apps/fluent-tester/src/TestComponents/Tooltip/TooltipDefault.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { View } from 'react-native';
+
+import { Button } from '@fluentui-react-native/experimental-button';
+import { Tooltip } from '@fluentui-react-native/tooltip';
+
+export const TooltipDefault: React.FunctionComponent = () => {
+  return (
+    <View>
+      <Tooltip content="Test" positioning="topCenter">
+        <Button>Test</Button>
+      </Tooltip>
+      <Tooltip content="Test 2">
+        <Button>Test 2</Button>
+      </Tooltip>
+      <Tooltip content="Test 3">
+        <Button>Test 3</Button>
+      </Tooltip>
+      <Tooltip content="Test 4">
+        <Button>Test 4</Button>
+      </Tooltip>
```

Comment:
Could you explain what Test 3 and Test 4 are testing for that's different from Test 2?

Reply:
I had included these tests for manually testing different variations and testing how it works when quickly hovering over several components. Removed the duplicate cases and intending to have more tests in follow-up PR so that it is easier to review.